extras/controllerClient/readme.md

```diff
@@ -0,0 +1,37 @@
+# NVDA Controller Client API 1.0 Documentation
+
+## Introduction
+
+This client API allows an application to communicate with NVDA, in order to do such things as speak text or braille a message.
+
+The client API is implemented as a dll (dynamic link library). The functions in this dll can be called from any programming language that supports looking up and calling of any symbol in a dll (such as ctypes in Python), or by linking to it for languages like C and C++.
+
+## What is Included?
+
+Note: The header and library files are built along with nvdaHelper. If you want to build them yourself, please see readme.txt in the NVDA source distribution for instructions. Alternatively, you can find here a [pre-built package](http://www.nvda-project.org/nvdaControllerClient/nvdaControllerClient_20100219.7z).
```

Comment:
This link still works, but is incredibly outdated. I'll accept this PR as it is, but it would be good if this link was updated. There are regular builds of this available from appveyor.